### ISSUE TITLE
Display a sum of story points in all Pipelines except Closed

### ISSUE THREAD

**Issue by drobiazko** [ZenHubIO/support#1359]
It would be extremly helpful to have a sum of all story points of currently visible stories in all the available pipelines except `Closed`. Currently panning or analyzing of stories in Zenhub without a calc is not possible.

![platform 2018-02-23 08-55-46](https://user-images.githubusercontent.com/464220/36583910-ddda4182-1877-11e8-87e4-ab8fa9afb8e0.png)

**Comment by kendramoroz** [comment]
Agreed! We actually have this very workflow in our backlog at the moment ✨ - We should have more to share very soon about when to expect something like this in the Board.
